bin/webpack-dev-server.js

```diff
@@ -27,13 +26,14 @@ const webpack = require('webpack');
 
 const options = require('./options');
 
-const { colors, status, version, bonjour, defaultTo } = require('./utils');
+const { colors, status, version, bonjour } = require('./utils');
 
 const Server = require('../lib/Server');
 
 const addEntries = require('../lib/utils/addEntries');
 const createDomain = require('../lib/utils/createDomain');
 const createLogger = require('../lib/utils/createLogger');
+const createSchema = require('../lib/utils/createSchema');
```

Comment:
Good idea, maybe better name for this are `createConfig` (avoid misleading between schema and configuration)?